src/components/util/util.js

```diff
@@ -222,3 +222,14 @@ export const isAValidURL = (str) => {
   return URL.canParse(str);
 };
 
+export const isAValidImageUrl = async (url) => {
+  try {
+    const response = await fetch(url);
```

Comment:
This is throwing CORS issue when tried with url like https://folio.org/wp-content/uploads/2023/08/folio-site-general-Illustration-social-image-1200.jpg